system/Commands/ListCommands.php

```diff
@@ -135,7 +135,7 @@ protected function describeCommands(array $commands = [])
 		// Pad each item to the same length
 		$names = $this->padArray($names, 2, 2);
 
-		for ($i = 0; $i < count($names); $i ++ )
+		for ($i = 0; $i < $countNames = count($names); $i ++ )
```

Comment:
Can you change this to assign $countNames outside of the loop:
    
    $countNames = count($names);
    for ($i = 0; $i <  $countNames; $i ++ )